doc/spec-files.rst

```diff
@@ -343,9 +343,32 @@ For example modify the spec file this way::
           exclude_binaries=...
           )
 
+One-File mode
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you are hoping to build a one file mode, the ``COLLECTION`` should be an empty tuple with its previous values absorbed by the ``EXE`` class. Your ``.exe`` file will still be in the dist folder with the name you specify here as the ``name``
```

Comment:
- Please break lines at 75.
- If `COLLECTION` is not passed any argument, you do not need to call it at all, as it plain does nothing. And as you could seen in the spec-file generated with `--onefile`, there is no COLLECTIOn at all.